src/NetAnalyzers/UnitTests/Microsoft.NetFramework.Analyzers/DoNotUseInsecureDtdProcessingXmlTextReaderConstructedWithNoSecureResolutionTests.cs

```diff
@@ -15,11 +15,11 @@ namespace Microsoft.NetFramework.Analyzers.UnitTests
 {
     public partial class DoNotUseInsecureDtdProcessingAnalyzerTests
     {
-        private static DiagnosticResult GetCA3075XmlTextReaderConstructedWithNoSecureResolutionCSharpResultAt(int line, int column)
-            => VerifyCS.Diagnostic().WithLocation(line, column).WithArguments(MicrosoftNetFrameworkAnalyzersResources.XmlTextReaderConstructedWithNoSecureResolutionMessage);
+        private static DiagnosticResult GetCA3075XmlTextReaderConstructedWithNoSecureResolutionCSharpResultAt(int line, int column) =>
+            VerifyCS.Diagnostic(DoNotUseInsecureDtdProcessingAnalyzer.RuleXmlTextReaderConstructedWithNoSecureResolution).WithLocation(line, column);
```

Comment:
📝 The `=>` generally appears on the second line when an expression-bodied member is wrapped. I'll let it slide in this pull request because currently there is no analyzer enforcement for the layout rule.

Reply:
I have found the two ways in the code and wasn't sure which one to apply. I have seen that operators are usually at the end (e.g. `&&`) so I have assumed the same but obviously I picked the wrong solution xD

Is such kind of styling rule something that shall go into `StyleCop`? Could we add `StyleCop` to this project to enforce some styles?

Reply:
Today there is no analyzer enforcement.

* Roslyn has [`CodeStyleOptions.OperatorPlacementWhenWrapping`](https://github.com/dotnet/roslyn/blob/2b8a9f75e88c18912d702b0aac2fb46f0ef04091/src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs#L119-L127) but it's only tied in to refactorings today
* StyleCop Analyzers has https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/748 but it's still just a proposal